Code/CryGame/Items/Weapons/VehicleWeapon.cpp

```diff
@@ -305,9 +305,12 @@ void CVehicleWeapon::UpdateFPView(float frameTime)
 //---------------------------------------------------------------------------
 void CVehicleWeapon::CheckForFriendlyAI(float frameTime)
 {
+	if (gEnv->bMultiplayer)
+		return;
```

Comment:
What about multiplayer AI in the future?

Reply:
Possibly, but this is just original code and not needed in mp for now :D